classes/class.ilMumieTaskGradeSync.php

```diff
@@ -205,4 +218,38 @@ private function getLatestGrade($xapi_grades)
         }
         return $latest_grade;
     }
+
+    public function wasGradeOverriden($user_id)
```

Comment:
This function should be moved to another class (something like `ilMumieTaskGradeOverrideService`). IlMumieTaskGradeSync is only responsible for retrieving data from MUMIE/Lemon servers and filtering them.

Btw: It's spelled 'Overridden' - with two Ds